### PR TITLE
Fix gtest 1.8.0 - compile gtest

### PR DESCRIPTION
**Pull request by hajgato**
Previous versions were installed as binary, `1.8.0` is a source version, so it have to be compiled.

### CODE REVIEW

File: easybuild/easyconfigs/g/gtest/gtest-1.8.0-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -8,13 +8,18 @@ description = "Google's framework for writing C++ tests on a variety of platform
 
 toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
 
+builddependencies = [
+    ('CMake', '3.10.2'),
+    ('binutils' '2.28'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/gtest/gtest-1.8.0-GCCcore-6.4.0.eb]: @hajgato missing `,`

Comment (hajgato) [reply easybuild/easyconfigs/g/gtest/gtest-1.8.0-GCCcore-6.4.0.eb]: @boegel fixed

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8b6683a649b9fce3b6fc4f81f7448d07 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2404.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7fefa13fd8ea0ebf850fa1069b37408e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**